python/ray/serve/_private/cluster_node_info_cache.py

```diff
@@ -67,3 +73,7 @@ def __init__(self, gcs_client: GcsClient):
 
     def get_draining_node_ids(self) -> Set[str]:
         return set()
+
+    def get_node_az(self, node_id: str) -> Optional[str]:
+        """Get availability zone of a node."""
+        return None
```

Comment:
I feel it will be easier if we return a string like "DEFAULT"/"UNKNOWN" here so the return type is not Optional.

It confused me a bit later on when I see Optional[str]: I don't know if it's because the replica is not started yet or we don't have az for the node.

Reply:
Makes sense, changed to return "UNKNOWN"!

Reply:
From other discussion, changed back to None because downstream we treat both the same way.